src/yardstick/tool/syft.py

```diff
@@ -43,9 +44,13 @@ def _install_from_installer(
         if not path:
             path = tempfile.mkdtemp()
             atexit.register(shutil.rmtree, path)
-        else:
-            if os.path.exists(os.path.join(path, "syft")):
-                tool_exists = True
+
+        if add_version_to_path:
```

Comment:
I don't understand the point of this variable. Why would we not want to add the version to the path? I thought that was how yardstick kept track of whether it needed to reinstall.

Reply:
removed